src/pages/case-study/doctorlink-setting-up-a-design-system.js

```diff
@@ -0,0 +1,578 @@
+import React, { Fragment } from 'react'
+import { graphql } from 'gatsby'
+import styled from 'styled-components'
+import breakpoint from 'styled-components-breakpoint'
+// import { Padding } from 'styled-components-spacing'
+import ReactMarkdown from 'react-markdown'
+// import generate from 'shortid'
+
+import { Grid, Row, Col } from '../../components/grid'
+import { SectionTitle, BodyPrimary } from '../../components/Typography'
+import CaseStudyHero from '../../components/Common/CaseStudyCards/CaseStudyHero'
+import Layout from '../../components/layout'
+// import { makeText } from '../../utils/makeText'
+import Head from '../../components/Common/Head'
+// import Image from '../../components/Common/Image'
+import VideoSection from '../../components/Common/VideoSection'
+// import SubtitleWithBody from '../../components/Common/SubtitleWithBody'
+// import Statement from '../../components/Common/Statement'
+import BlueBackground from '../../components/Common/BlueBackground'
+import GreyBackground from '../../components/Common/GreyBackground'
+import CaseStudyPreview from '../../components/Common/CaseStudyCards/CaseStudyPreview'
+
+// const PaddingBottomGrid = styled(Grid)`
+//   padding-bottom: ${({ theme }) => theme.space[7]};
+// `
+
+const BlockRow = styled(Row)`
+  padding-top: ${({ theme, topIndex }) =>
+    topIndex ? theme.space[topIndex] : null};
+  padding-bottom: ${({ theme, bottomIndex }) =>
+    bottomIndex ? theme.space[bottomIndex] : null};
+  flex-direction: ${({ rowReverse }) => `row${rowReverse ? '-reverse' : null}`};
```

Comment:
because flex-direction defaults to  `row` you can you just conditionally add reverse

`${({ rowReverse })=> rowReverse ? 'flex-direction: 'row-reverse' :  null}`

Reply:
You sure of that? I just tried
`${({ rowReverse }) => (rowReverse ? 'flex-direction: row-reverse' : null)}` and it does not seem to work